system-tests/__snapshots__/record_spec.js

```diff
@@ -2890,88 +2890,6 @@ exports['e2e record capture-protocol enabled passing retrieves the capture proto
   Recorded Run: https://dashboard.cypress.io/projects/cjvoj7/runs/12
 
 
-`
```

Comment:
Was this one expected to be removed? I didn't see any changes to record_spec.js.

Reply:
I don't either, but it also doesn't match any tests in record_spec. I'm guessing it was an obsolete snapshot that didn't get removed on a previous merge.

Reply:
Sounds good. Good to clean it up then.